README.md

```diff
@@ -1,8 +1,313 @@
 # FuseML Core
 
-This repository contains the FuseML APIs definitions and core service.
+This repository contains the FuseML APIs definitions and core service. For the general information about FuseML project, check the [main repository page](https://github.com/fuseml/fuseml).
+
+## Installation
+
+* If you are using [fuseml installer](https://github.com/fuseml/fuseml), the core service is installed into your Kubernetes cluster along with other components and the command line client is downloaded to your working directory. It is recommended to copy the client to the location with other executables, e.g. `/usr/local/bin/` on the Linux systems.
+
+
+* To download pre-built client and server components, check the [releases page](https://github.com/fuseml/fuseml-core/releases).
+
+
+* To build the latest version directly from sources:
+
+  * Make sure you have `go` installed, at least version 1.16.
+
+  * Download [protocol buffers](https://github.com/protocolbuffers/protobuf). Use the release page for [version 3.15.7](https://github.com/protocolbuffers/protobuf/releases/tag/v3.15.7) and install only the `protoc` binary.
+
+    On Linux, you can proceed this way:
+    ```bash
+    wget https://github.com/protocolbuffers/protobuf/releases/download/v3.15.7/protoc-3.15.7-linux-x86_64.zip
+    unzip protoc-3.15.7-linux-x86_64.zip
+    sudo cp -a bin/protoc /usr/local/bin/
+    ```
+
+  * Clone `fuseml-core` repository
+    ```bash
+    git clone git@github.com:fuseml/fuseml-core.git
+    cd fuseml-core
+    ```
+
+  * Install go dependencies
+    ```bash
+    make deps
+    ```
+
+  * Generate server, client and CLI code and build the binaries.
+    ```bash
+    make all
+    ```
+    This will produce server (`bin/fuseml_core`) and command line client (`bin/fuseml`) binaries. The client binary is compiled for your current architecture.
+
+
+## Usage
+
+* Run the server localy
+
+  After successful installation using [fuseml-installer](https://github.com/fuseml/fuseml), `fuseml-core` server is already running in the Kubernetes cluster.However, for developmnent and testing purposes it is possible to run it locally on your machine by executing `bin/fuseml_core`. It's necessary to set some environment variables first.
```

Comment:
Missing space in `...cluster.However...`

`run it locally on your machine` sounds a bit redundant for me.

Suggestion: mention the requirements (setting the env variables) before running the binary, so it follows the sequence. Something like `... its possible to run it locally by setting the following env variables... and executing `bin/fuseml_core`

Reply:
Good idea.